src/index.ts

```diff
@@ -216,20 +216,23 @@ export const getGPUTier = async ({
     }
 
     const tokenizedRenderer = tokenizeForLevenshteinDistance(renderer);
-    // eslint-disable-next-line prefer-const
-    let [gpu, , , , fpsesByPixelCount] =
+
```

Comment:
This is where their old code was going wrong where it was incorrectly retrieving the placement of where the fpsesByPixelCount should be - so for now to support both data shapes we just assume its the last entry in the array which seems to be accurate between how it used to look and now - if this ever changes again though our test should pick it up